getAzureReposBasedOnlatestCommit [Updated]/getAzureReposBasedOnlatestCommitInformation.py

```diff
@@ -0,0 +1,158 @@
+import requests
+import os
+import pandas as pd
+import logging
+
+## get the current directory
+c_dir = os.path.dirname(os.path.realpath(__file__))
+
+## creating error.log file
+with open(f'{c_dir}/error.log', 'w'):
+    pass
+
+## configure logging file
+logging.basicConfig(filename=f'{c_dir}/error.log',level=logging.ERROR,format='%(asctime)s - %(levelname)s - %(message)s')
+
+def getAuth(ado_pat) :
+    ## header config
+    headers = {"Content-Type": "application/json"}
+    ## for Basic Auth
+    auth = ('', ado_pat)
+    return (headers,auth)
+def getErrorLogs() :
+    repos_id =[]
+    with open(f'{c_dir}/error.log', 'r') as file:
+        error_log_contents = file.read()
+        error_repos = error_log_contents.split('\n')
+        for error_repos in error_repos :
+            repos_id.append(error_repos.split(',')[-1].split(': ')[-1])
+        pass
```

Comment:
what is this pass doing please add comment.

Reply:
It leaves the rest execution in that block

Reply:
after pass there is only return statement and then the next function so after pass what is it executing?

Reply:
it execute the return statement